cmake/std/atdm/apps/empire/EMPIRETrilinosEnables.cmake

```diff
@@ -1,4 +1,21 @@
+IF ("$ENV{ATDM_CONFIG_BUILD_TYPE}" STREQUAL "DEBUG")
+	ATDM_SET_ENABLE(PanzerAdaptersSTK_CurlLaplacianExample-ConvTest-Quad-Order-4_DISABLE ON)
+	ATDM_SET_ENABLE(PanzerAdaptersSTK_MixedPoissonExample-ConvTest-Hex-Order-3_DISABLE ON)
+ENDIF()
+
+IF (ATDM_USE_CUDA)
+	ATDM_SET_ENABLE(Sacado_ENABLE_HIERARCHICAL_DFAD ON)
+ENDIF()
+
 ATDM_SET_ENABLE(MueLu_ENABLE_Kokkos_Refactor_Use_By_Default ON)
+ATDM_SET_ENABLE(Panzer_ENABLE_EXAMPLES ON)
+ATDM_SET_ENABLE(Panzer_ENABLE_TESTS ON)
+ATDM_SET_ENABLE(Trilinos_ENABLE_Panzer ON)
```

Comment:
The set of package enables is already handled in the file:

* https://github.com/trilinos/Trilinos/blob/develop/cmake/std/atdm/apps/empire/EMPIRETrilinosPackagesList.cmake

which already gets included in this file. 

Therefore, we can remove them from this file.